data/net/minecraft/client/renderer/BlockEntityWithoutLevelRenderer.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/client/renderer/BlockEntityWithoutLevelRenderer
+	METHOD <init> (Lnet/minecraft/client/renderer/blockentity/BlockEntityRenderDispatcher;Lnet/minecraft/client/model/geom/EntityModelSet;)V
+		ARG 1 blockEntityRenderDispatcher
+		ARG 2 entityModelSet
+	METHOD renderByItem (Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/client/renderer/block/model/ItemTransforms$TransformType;Lcom/mojang/blaze3d/vertex/PoseStack;Lnet/minecraft/client/renderer/MultiBufferSource;II)V
+		COMMENT Override this to implement your own rendering logic for your item
```

Comment:
I don't like this comment, as this defines mod platform specific behavior. If this is a useful comment, it should be phrased in how minecraft uses it (e.g. "This is overriden in places X, Y, Z for reasons A, B, C").

As an example, a mod platform might indeed *not* say that this is how to implement rendering logic, and have their own API or system.

Reply:
Hmm, I guess you are right. Then I think it is best to just remove the comment altogether!